### ISSUE TITLE
[Icon] Add aria-label and role="img" to icon SVG

### ISSUE THREAD

**Issue by lukbrt** [ant-design/ant-design#28782]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Antd SVG icons are wrapped inside span which has defined aria-label and role="img" attributes, but when I try to validate my page with WCAG tools (e.g. ARC Toolkit) it shows the following issues:
- noImageRole
- svgMissingAriaLabelledby

### What does the proposed API look like?
The aria-label and role attributes should be defined both at the  <span class="anticon" /> wrapper as well as the SVG tag itself or there should be a way to pass custom custom props to the SVG tag. E.g.
<CalendarOutlined svgProps={
   ariaLabel: "my aria label"
   role: "img"
} />

**Comment by zombieJ** [comment]
I believe it's a false positive warning. When parent node is mark role as `img`. Whatever child elements are, it will be trade as a single image.

ref: https://www.w3.org/TR/wai-aria/#img
